com.rioterm.Rio.yml

```diff
@@ -0,0 +1,54 @@
+---
+app-id: com.rioterm.Rio
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+
+finish-args:
+  - --device=dri
+  - --filesystem=xdg-config/rio
```

Comment:
why is this needed?

Reply:
This is where it stores the configuration. Either `--filesystem` or `--persist` is needed otherwise it loses the configuration each reopen

Reply:
you mean it ignores `XDG_CONFIG_HOME` ? The it should use `--persist`.

Reply:
so it does ignore it.

Reply:
Actually, `--persist` does not work. The config directory is empty on each app start.

Reply:
@hfiguiere you are correct, it does not use `$XDG_CONFIG_HOME`. I will submit a PR upstream fixing it

Reply:
if you expect something in `~/.config` then that mean it doesn't respect `XDG_CONFIG_HOME`. Also persist does NOT expect the same value as for filesystem.

Reply:
Will be fixed upstream with the next release 😉

Reply:
so either this submission wait for the next release, or you carry over the patch so that this works.

Reply:
I will wait for the new release, which should happen as soon as all requirements from here are pushed :)

Reply:
Done!